src/main/java/com/mcmoddev/bot/command/CommandXY.java

```diff
@@ -0,0 +1,34 @@
+package com.mcmoddev.bot.command;
+
+import com.mcmoddev.bot.util.Utilities;
+import sx.blah.discord.handle.obj.IMessage;
+import sx.blah.discord.util.EmbedBuilder;
+
+public class CommandXY implements Command
+{
+    @Override
+    public void processCommand(IMessage message, String[] params)
+    {
+        final StringBuilder builder = new StringBuilder();
```

Comment:
Any reason for the string builder here? You only really need this if you're programatically joining data together and want to lower memory. In this case, where the data never changes, you don't need this.